modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/helper/ValidationHelper.java

```diff
@@ -31,6 +31,21 @@
 @Component(service = ValidationHelper.class)
 public class ValidationHelper {
 
+	public boolean isValidAPIApplicationCompanyId(
+		long objectEntryId, long companyId) {
```

Comment:
Arguments must be sorted

Reply:
@sergiojimcos done ! https://github.com/liferay-headless/liferay-portal/pull/1896/commits/8aac21cd6b455f1553941c25a4981d82ef20102b